### PR TITLE
k4dirstat: init at 3.2.1

### PR DESCRIPTION
**Pull request by raboof**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/k4dirstat/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    sha256 = "15xjb80jq6vhzvzx4l341f40d8a23w1334qh6cczqm9adfnzycp7";
+  };
+  meta = {
+    license = with lib.licenses; [ gpl2 ];
+    description = "A small utility program that sums up disk usage for directory trees";
+    maintainers = [ lib.maintainers.raboof ];
+  };
+  nativeBuildInputs = [ extra-cmake-modules kdoctools kxmlgui ];
+  buildInputs = [ kiconthemes kio kjobwidgets ];
+  propagatedBuildInputs = [ ];
```
Comment (IvarWithoutBones) [inline pkgs/applications/misc/k4dirstat/default.nix:28]: This defaults to being empty, no? You can just remove it.

File: pkgs/applications/misc/k4dirstat/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    repo = pname;
+    rev = version;
+    sha256 = "15xjb80jq6vhzvzx4l341f40d8a23w1334qh6cczqm9adfnzycp7";
+  };
+  meta = {
+    license = with lib.licenses; [ gpl2 ];
+    description = "A small utility program that sums up disk usage for directory trees";
+    maintainers = [ lib.maintainers.raboof ];
+  };
+  nativeBuildInputs = [ extra-cmake-modules kdoctools kxmlgui ];
```
Comment (IvarWithoutBones) [inline pkgs/applications/misc/k4dirstat/default.nix:26]: Shouldn't `kdoctools` and `kxmlgui` be regular `buildInputs`?

**Review by IvarWithoutBones — CHANGES_REQUESTED** [review]
Some spacing + meta stuff seems odd, heres a suggestion:
```nix
{ stdenv
, mkDerivation
, fetchFromGitHub
, extra-cmake-modules
, kdoctools
, kiconthemes
, kio
, kjobwidgets
, kxmlgui
}:

mkDerivation rec {
  pname = "k4dirstat";
  version = "3.2.1";

  src = fetchFromGitHub {
    owner = "jeromerobert";
    repo = pname;
    rev = version;
    sha256 = "15xjb80jq6vhzvzx4l341f40d8a23w1334qh6cczqm9adfnzycp7";
  };

  nativeBuildInputs = [ extra-cmake-modules kdoctools kxmlgui ];
  buildInputs = [ kiconthemes kio kjobwidgets ];

  meta = with stdenv.lib; {
    homepage = "https://github.com/jeromerobert/k4dirstat";
    description = "A small utility program that sums up disk usage for directory trees";
    license = licenses.gpl2;
    maintainers = [ maintainers.raboof ];
    platforms = platforms.linux;
  };
}
```

**Comment by raboof** [comment]
Updated, thanks!

**Review by IvarWithoutBones — APPROVED** [review]
LGTM :)

**Comment by nh2** [comment]
Result of `nixpkgs-review pr 91345` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
<br>- k4dirstat
</details>

**Comment by nh2** [comment]
GUI works for me.

**Pull request merged**